projects/aca-testing-shared/src/utilities/utils.ts

```diff
@@ -71,6 +72,11 @@ export const isPresentAndDisplayed = async (element: ElementFinder): Promise<boo
   }
 };
 
+
+export const getTitleElementText = (fixture: ComponentFixture<any>): string => {
```

Comment:
This library is for e2e mainly as here you have references to protractor. I think for now it will be fine if you put these functions to spec files directly like you did for ADF